src/datasets/load.py

```diff
@@ -1671,6 +1676,8 @@ def load_dataset(
             Multiprocessing is disabled by default.
 
             <Added version="2.7.0"/>
+        storage_options (`dict`, *optional*, defaults to `None`):
+            Key/value pairs to be passed on to the dataset file-system backend, if any.
```

Comment:
I'd keep this feature experimental until we have a proper data files resolution based on fsspec (e.g. request_etag and is_remote_url don't really support fsspec) as well as some tests:
```suggestion
            **Experimental**. Key/value pairs to be passed on to the dataset file-system backend, if any.
```